wiki/Article_styling_criteria/Formatting/de.md

```diff
@@ -0,0 +1,782 @@
+
+### Maximale Bildbreite
+
+Die maximale erlaubte Bildbreite der Webseite ist die Breite des Artikels. Bilder sollten nicht breiter als 800 Pixel sein.
+
+### Bilder mit Anmerkungen versehen
+
+Wenn Bilder mit Anmerkungen versehen werden, benutze bitte *Torus Regular*. Nutze *Microsoft YaHei* für chinesische, koreanische und japanische Schriftzeichen.
+
+Bildern mit Anmerkungen zu versehen sollte allerdings vermieden werden, da es schwierig für Übersetzer (und andere Bearbeiter) ist, diese zu editieren.
+
+#### Bilder mit Anmerkungen übersetzen
+
+Bei der Übersetzung von Bildanmerkungen, sollte die lokalisierte Version im selben Verzeichnis wie die originale Version (also die englische) platziert werden. Der Dateiname für das lokalisierte Bild muss mit dem Namen der originalen Version starten, danach ein Bindestrich und im Anschluss das Locale-Kürzel (in Großbuchstaben). Siehe folgendes Beispiel:
+
+- `hardrock-mod-vs-easy-mod.jpg` für Englisch
+- `hardrock-mod-vs-easy-mod-DE.jpg` für Deutsch
+- `hardrock-mod-vs-easy-mod-ZH-TW.jpg` für traditionelles Chinesisch
+
+### Screenshots des Spielverlaufs
+
+Alle Screenshots des Spielverlaufs müssen in der Updatequelle Stable gemacht werden, außer es ist ein spezifisches Feature, welches nicht in Stable vorhanden ist. Du solltest die Screenshot-Funktion des Spiels (`F12`) nutzen.
+
+#### Einstellungen des Spiel-Clients
+
+*Tipp: Wenn du deine aktuellen Einstellungen für das Wiki nicht ändern möchtest, dann kannst du die Datei `osu!.<ComputerUser>.cfg` aus deinem osu!-Ordner entfernen und später wieder hinzufügen.*
+
+Du musst diese Einstellungen im Spiel-Client haben, bevor du einen Screenshot erstellst (Einstellungen, die unten nicht genannt werden, sollten auf ihrem Standardwert stehen):
+
+- Sprache wählen: `Englisch`
+- Metadaten in Originalsprache: `Aktiviert`
+- Bildschirmauflösung: `1280x720`
+- Vollbildmodus: `Deaktiviert`
+- Parallaxeneffekt: `Deaktiviert`
+- Menütipps: `Deaktiviert`
+- Seasonale Hintergünde: `Niemals`
+- Tasten-Overlay: `Aktiviert`
+- Aktueller Skin: `Standard` (erste Option)
+
+*Hinweis für Übersetzer: Wenn du einen Artikel mit Screenshots des Spiels übersetzt, dann darf die Sprache des Spiel-Clients in der übersetzten Sprache sein.*
+
+### Bilderlinks
+
+Bilder dürfen nicht Teil eines Linktextes sein.
+
+Flaggensymbole neben Links zu Benutzern müssen separat vom Linktext sein. Siehe folgendes Beispiel:
+
+```markdown
+![][flag_AU] [peppy](https://osu.ppy.sh/users/2)
+```
+
+### Flaggensymbole
+
+*Für eine Liste der Flaggensymbole, siehe: [Issue \#328](https://github.com/ppy/osu-wiki/issues/328)*
+
+Die Flaggensymbole benutzen eine Zwei-Buchstaben-Kodierung (alle in Großschreibweise) und enden in `.gif`. Benutze beim Hinzufügen eines Symbols in der Linie dieses Format:
```

Comment:
```suggestion
Die Flaggensymbole benutzen eine Zwei-Buchstaben-Kodierung (alle in Großschreibweise) und enden in `.gif`. Benutze beim Inline-Hinzufügen eines Symbols dieses Format:
```
Might as well use `inline` here, too. Or maybe change `in der Linie` to `in der Zeile` or `im Text` even. I'm still not really happy with my suggestions either, but I can't think of anything better ^^

Reply:
I think Inline is fine for the time being. Sometimes, there isn't any good word for translation.